source/usr/lib/mcvirt/rpc/pyro_object.py

```diff
@@ -0,0 +1,76 @@
+"""Base class for providing Pyro-based methods for objects"""
+# Copyright (c) 2016 - I.T. Dev Ltd
+#
+# This file is part of MCVirt.
+#
+# MCVirt is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# MCVirt is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with MCVirt.  If not, see <http://www.gnu.org/licenses/>
+
+import Pyro4
+
+
+class PyroObject(object):
+    """Base class for providing Pyro-based methods for objects"""
+
+    @property
+    def _is_pyro_initialised(self):
+        """Determine if object is registered with the Pyro deamon"""
+        if '_pyroDaemon' not in self.__dict__.keys():
+            return False
+        else:
+            return True
```

Comment:
You could have
``return '_pyroDaemon' in self.__dict__.keys()``
to save a few lines